fr.po

```diff
@@ -4379,11 +4379,11 @@ msgstr ""
 
 #. heading
 msgid "Pick Topic"
-msgstr "Choix du thème"
+msgstr "Thème"
```

Comment:
was this a problem in the UI? the previous string doesn't seem too long and might be clearer to the player?